docs/sources/mimir/release-notes/v2.6.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Grafana Mimir version 2.6 release notes"
+menuTitle: "V2.6 release notes"
+description: "Release notes for Grafana Mimir version 2.6"
+weight: 300
+---
+
+# Grafana Mimir version 2.6 release notes
+
+Grafana Labs is excited to announce version 2.6 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Lower memory usage in store-gateway by streaming series results**
+  The store-gateway can now stream results back to the querier instead of buffering them. This is expected to greatly reduce peak memory consumption while keeping latency the same. This is still an **experimental feature** and is subject to a changing API and instability.
+
+- **Improved stability in store-gateway by removing mmap usage**
+  The store-gateway can now use an alternate code path to read index-headers that does not use memory mapped files. This is expected to improve stability of the store-gateway. This is still an **experimental feature** and is subject to a changing API and instability.
+
+- **Alertmanager Webex support**
+  Alertmanager can now use Webex to send alerts.
+
+### Helm chart improvements
+
+TBD
+
+## Important changes
```

Comment:
Can you also mention this? We want people to stop using `-store.max-query-length` and use `-querier.max-partial-query-length` instead.

> * [CHANGE] Querier: Introduce `-querier.max-partial-query-length` to limit the time range for partial queries at the querier level and deprecate `-store.max-query-length`. #3825

Reply:
Sure but I think we need further work to actually deprecate this setting. Normally when we deprecate something we use `flagext.DeprecatedFlag` and immediately start ignoring the flag and refactor all code to use the new flag. Shouldn't be we doing that here?

Reply:
`-store.max-query-length` has already been removed from the doc. It's still parsed as a fallback and we should have updated all places in the documentation mentioning it.